scripts/validate_docstrings.py

```diff
@@ -543,7 +543,7 @@ def validate_pep8(self):
         application = flake8.main.application.Application()
         application.initialize(["--quiet"])
 
-        with tempfile.NamedTemporaryFile(mode='w') as file:
+        with tempfile.NamedTemporaryFile(mode='w', encoding='utf-8') as file:
```

Comment:
If you can revert this I think the rest looks good

Reply:
Oh, i again commit this line. I should create github issue with coresponding PR for that.